tests/core/test_bungiesearch.py

```diff
@@ -100,6 +137,9 @@ def test_no_results(self):
         '''
         self.assertEqual(list(Article.objects.search.query('match', _all='nothing')), [], 'Searching for "nothing" did not return an empty list on iterator call.')
         self.assertEqual(Article.objects.search.query('match', _all='nothing')[:10], [], 'Searching for "nothing" did not return an empty list on get item call.')
+        
+        self.assertEqual(list(User.objects.search.query('match', _all='nothing')), [], 'Searching for "nothing" did not return an empty list on iterator call.')
+        self.assertEqual(list(User.objects.search.query('match', _all='nothing')), [], 'Searching for "nothing" did not return an empty list on iterator call.')
```

Comment:
Line 142 is identical to line 141. You want to test the list (`__iter__` function) and the slice (`__getitem__` function) operations here so one should be `list(...)` and the other `()[x:y]` (cf. lines 138 and 139).